src/components/Input/Input.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import "./Input.css";
+
+type InputProps = {
+  value: string;
+  placeholder: string;
```

Comment:
Необязательный пропс

```suggestion
  placeholder?: string;
```